gnmi_server/server.go

```diff
@@ -43,7 +43,9 @@ type Server struct {
 	// comes from a master controller.
 	ReqFromMaster func(req *gnmipb.SetRequest, masterEID *uint128) error
 	masterEID     uint128
+	gnoi_system_pb.UnimplementedSystemServer
```

Comment:
what does this mean?

Reply:
To satisfy SystemServer interface requirements for this gNOI version, all implementations must embed UnimplementedSystemServer (added to gNOI for forward compatibility. For reference: https://github.com/openconfig/gnoi/blob/349b3dcd04ecf78d740a25f17f983a8c8cd56e73/system/system_grpc.pb.go#L226)

Reply:
Please add comment here

Reply:
Added comment